types/index.d.ts

```diff
@@ -183,6 +186,9 @@ export class request {
 
   /** @deprecated */
   session: (key: string) => any;
+
+  /** Returns the request CanFulfillIntent */
+  getCanFulfillIntent: () => object;
```

Comment:
Can we strongly type this? You should be able to specify that it returns a `canFulfillIntent`.